lighthouse-core/test/lib/i18n/i18n-test.js

```diff
@@ -98,6 +113,44 @@ describe('i18n', () => {
     });
   });
 
+  describe('#registerLocaleData', () => {
+    it('installs new locale strings', () => {
+      const localeData = {
+        'lighthouse-core/test/lib/i18n/i18n-test.js | testString': {
+          'message': 'en-XZ cuerda!'
+        }
+      };
+      i18n.registerLocaleData('en-XZ', localeData);
+
+      const UIStrings = {testString: 'en-US string!'};
+      const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+      const formattedStr = i18n.getFormatted(str_(UIStrings.testString), 'en-XZ');
+      expect(formattedStr).toEqual('en-XZ cuerda!');
+    });
+
+    it('overwrites existing locale strings', () => {
+      const filename = 'lighthouse-core/audits/is-on-https.js';
+      const UIStrings = require('../../../../' + filename).UIStrings;
+      const str_ = i18n.createMessageInstanceIdFn(filename, UIStrings);
+
+      // To start with, we get back the intended string..
+      const formattedStr = i18n.getFormatted(str_(UIStrings.title), 'es-419');
+      expect(formattedStr).toEqual('Usa HTTPS');
+
+      // Now we declare and register the new string...
+      const localeData = {
+        'lighthouse-core/audits/is-on-https.js | title': {
+          'message': 'es-419 uses https!'
+        }
+      };
+      i18n.registerLocaleData('es-419', localeData);
+
+      // And confirm that's what is returned
+      const formattedStr2 = i18n.getFormatted(str_(UIStrings.title), 'es-419');
+      expect(formattedStr2).toEqual('es-419 uses https!');
```

Comment:
one more assertion that some other string that used to be there is removed? would be nice to clarify in the tests that it's not merging or something it's just totally replacing